### DISCUSSION TITLE
Support for AES encryption in Crypto Library

### DISCUSSION THREAD

**Discussion by debianmaster** [open-policy-agent/community#553] [OPA and Rego]
I have a use case where i need to encryption url params as part of rego code.  is there existing library for doing AES encryption in rego language?

i.e.   encyrption support in  https://www.openpolicyagent.org/docs/latest/policy-reference/#crypto

**Comment by anderseknert** [comment]
Hi there! No existing support, but  should be a fairly easy addition to the language. I'd suggest opening a feature request issue in OPA for this, describing your use case.
